architect.yml

```diff
@@ -1,32 +1,36 @@
 name: nuxt
-description: A starter project showing how to make use of JSX with Typescript in Nuxt.js
+description: A starter project showing how to make use of Typescript in Nuxt.js
 keywords:
   - nuxt.js
   - typescript
-  - jsx
-  - tsx
+  - vue
+
+dependencies:
+  node-rest-api: latest
 
 secrets:
   APP_NAME:
     description: Name of the application to use in page metadata
-    default: Nuxt TSX Starter
+    default: Nuxt Starter
 
 services:
   app:
+    command: npm run start
     build:
       context: ./
       args:
         NODE_ENV: production
     environment:
-      BASE_URL: ${{ services.app.interfaces.main.ingress.url }}
       APP_NAME: ${{ secrets.APP_NAME }}
+      APP_API_ADDR: ${{ dependencies['node-rest-api'].services.api.interfaces.main.url }}
```

Comment:
Same deal as [this comment](https://github.com/architect-templates/nuxt/pull/11/files#r1056623790) - don't want the user to touch dependencies until after they get their project running